1.8/usage/service-discovery/dns-overview.md

```diff
@@ -0,0 +1,68 @@
+---
+post_title: DNS Quick Reference
+nav_title: DNS Reference
+menu_order: 20
+---
+
+We've got a lot going on when it comes to DNS and this article aims to
```

Comment:
Suggest rewrite: `DC/OS offers a lot flexibility and extensibility for DNS. This quick reference provides a summary of the available options.`